### ISSUE TITLE
False positive unused return value warning for String functions

### ISSUE THREAD

**Issue by weisslj** [rrrene/credo#539]
### Environment

* Credo version (`mix credo -v`): 0.9.2
* Erlang/Elixir version (`elixir -v`): Erlang/OTP 20.3.4, Elixir 1.6.4 (compiled with OTP 19)
* Operating system: Debian GNU/Linux 9 (stretch)

### What were you trying to do?

Run credo on our project, minimal example:

```
$ mix credo --format=flycheck --read-from-stdin <<EOF
> defmodule Buggy do
>   @moduledoc false
>   def parse(str) do
>     String.to_integer(str)
>   rescue
>     ArgumentError -> String.to_float(str)
>   end
> end
> EOF
```

### Expected outcome

No warning.

### Actual outcome

stdin:6:22: W: There should be no unused return values for String functions.

**Comment by rrrene** [comment]
Thx for reporting! :+1:

**Comment by rrrene** [comment]
@weisslj 
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by weisslj** [comment]
Thank you, I tested the current master, this solves our issue!
